airflow-core/tests/unit/utils/test_db.py

```diff
@@ -74,6 +75,9 @@ def test_database_schema_and_sqlalchemy_model_are_in_sync(self):
         # test FAB models
         for table_name, table in FABDBManager.metadata.tables.items():
             all_meta_data._add_table(table_name, table.schema, table)
+        # test Human-in-the-loop models
+        for table_name, table in HITLProviderDBManager.metadata.tables.items():
+            all_meta_data._add_table(table_name, table.schema, table)
         # create diff between database schema and SQLAlchemy model
         mctx = MigrationContext.configure(
```

Comment:
Hey @jedcunningham, I'm having some trouble with this test and would appreciate your insight on it. `all_meta_data` seems to include the new tables I added in this PR, while `mctx` does not. Is there anything additional I need to do on the migration side?

Reply:
It's using a plugin and will be initialized when external database managers are set. I thought I was doing something similar here?